layouts/partials/footer/footer.html

```diff
@@ -3,7 +3,7 @@
     <div class="row">
       <div class="col-lg-8 order-last order-lg-first">
         <ul class="list-inline">
-          <li class="list-inline-item">{{ .Site.Params.footer | safeHTML }}</li>
+          <li class="list-inline-item">Sponsored by <a href="">BluBracket</a></li>
```

Comment:
I'd preserve the `{{ .Site.Params.footer | safeHTML }}` to keep the template versatile